### PR TITLE
Adding static CSS file and updating repo.

### PR DESCRIPTION
**Pull request by EdwinGuzman**
Moved from [Bitbucket](https://bitbucket.org/NYPL/dgx-global-footer/src/master/) to Github. Yay!

But also updated the repo with a changelog and updated readme. Updated the SCSS from [dgx-react-footer](https://bitbucket.org/NYPL/dgx-react-footer/src) and added a makefile to compile it to CSS. This is just to make it easier for others to copy/paste the HTML and CSS into their project (typically for third party sites that want to use the NYPL footer).

Demo (also found in the readme): https://codepen.io/edwinguzman/pen/qYXzPx

### CODE REVIEW

File: styles/_mixins.scss [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+    clear: both;
+  }
+}
+
+/* VALUE CALCULATION MIXIN
+ * ---------------------
+ * Usage: Makes the properties of an element
+ * can be defined by both percentage
+ * and pixels.
+*/
```
Comment (gkallenberg) [inline styles/_mixins.scss:100]: Params def here ??

File: README.md [hunk 2]
```diff
@@ -1,22 +1,25 @@
...
 ### Version
+1.0.0
 
+### Usage
+Copy the static HTML in `footer.html` and the CSS in `build/footer.css` and add it to your project.
+
+### Build
+The SCSS is copying over from [dgx-react-footer](https://bitbucket.org/NYPL/dgx-react-footer/src/master/). Feel free to use that if you'd like to get the source styles and related methods.
+
+You can update the SCSS and compile new CSS by running `make` at the top level of this repository. This assumes you already have sass installed on your machine.
```
Comment (ktp242) [inline README.md:14]: probably offering a command line there to show how to run `make`

Comment (EdwinGuzman) [reply README.md:14]: The command is just `make`. Seems like a bit too much for a single word command.

**Review by ktp242 — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (ktp242) [reply README.md:14]: Just `make`? lol OK then

**Review by gkallenberg — APPROVED** [review]
just added a comment about code comments.

File: styles/_mixins.scss (hunk 1 above)
Comment (EdwinGuzman) [reply styles/_mixins.scss:100]: Ah, that's the esdoc convention for javascript but we don't have one for CSS.

**Comment by EdwinGuzman** [comment]
Merging!

**Pull request merged**